code/app/src/main/AndroidManifest.xml

```diff
@@ -15,13 +15,18 @@
         <activity
             android:name="com.adobe.marketing.edge.identity.app.MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar">
+            android:theme="@style/AppTheme.NoActionBar"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="com.google.android.gms.ads.APPLICATION_ID"
+            android:value="@string/gms_ads_app_id"/>
```

Comment:
Without a valid  application id,  getting RuntimeException error
2022-05-03 14:28:32.407 32417-32417/com.adobe.marketing.edge.identity.app E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.adobe.marketing.edge.identity.app, PID: 32417
    java.lang.RuntimeException: Unable to get provider com.google.android.gms.ads.MobileAdsInitProvider: java.lang.IllegalStateException: 
    
    ******************************************************************************
    * Invalid application ID. Follow instructions here:                          *
    * https://googlemobileadssdk.page.link/admob-android-update-manifest         *
    * to find your app ID.                                                       *
    ******************************************************************************
    
    
        at android.app.ActivityThread.installProvider(ActivityThread.java:7244)
        at android.app.ActivityThread.installContentProviders(ActivityThread.java:6780)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6697)
        at android.app.ActivityThread.access$1300(ActivityThread.java:237)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: java.lang.IllegalStateException: 


Although it is expected. But getting  a  fatal exception error doesn't  give a good experience for the user.  How can we redirect the user to get setup their Application ID before running the app.

Reply:
@timkimadobe Discussed this with Emilia, she suggested to add a step for  setup secret.xml with admob application id in the sample app [readme](https://github.com/adobe/aepsdk-edgeidentity-android#development)  -> Run demo application.
Once users go through the readme, then they should already have the id setup  and won't see the exception error.

Reply:
Will take this update in a separate PR.